src/project/ProjectModel.js

```diff
@@ -59,6 +59,13 @@ define(function (require, exports, module) {
     var _exclusionListRegEx = /\.pyc$|^\.git$|^\.gitmodules$|^\.svn$|^\.DS_Store$|^Thumbs\.db$|^\.hg$|^CVS$|^\.hgtags$|^\.idea$|^\.c9revisions$|^\.SyncArchive$|^\.SyncID$|^\.SyncIgnore$|\~$/;
 
     /**
+     * @public
```

Comment:
And there should be a preference to easily extend this list. But that's a different story.